package.json

```diff
@@ -53,10 +53,10 @@
     "@aics/frontend-insights": "~0.2.3",
     "@aics/frontend-insights-plugin-amplitude-node": "~0.2.3",
     "@ant-design/icons": "~4.7.0",
-    "@types/spark-md5": "^3.0.2",
     "@types/react-beautiful-dnd": "~13.1.2",
     "@types/react-virtualized-auto-sizer": "~1.0.0",
     "@types/react-window": "~1.8.3",
+    "@types/spark-md5": "^3.0.2",
```

Comment:
Oh I didn't notice this when this was added, but this isn't using the same version restrictions as the others `^` vs `~`. Could you switch this to be `~3.0.2` along with `spark-md5` below on line 83? 
`^` means it will auto-upgrade to patch & minor versions of the same major version.
`~` means it will auto-upgrade to patch version of the same major and minor version (more restrictive)